code/datums/chatmessage.dm

```diff
@@ -184,14 +192,14 @@
   * * italics - Vacuum and other things
   * * size - Size of the message
   */
-/mob/proc/create_chat_message(atom/movable/speaker, raw_message, italics=FALSE, size)
+/mob/proc/create_chat_message(atom/movable/speaker, raw_message, italics=FALSE, size, symbol)
 
 	if(isobserver(src))
 		return
 
 
 	// Display visual above source
-	new /datum/chatmessage(raw_message, speaker, src, italics, size)
+	new /datum/chatmessage(raw_message, speaker, src, italics, size, CHAT_MESSAGE_LIFESPAN, symbol)
```

Comment:
```suggestion
	new /datum/chatmessage(raw_message, speaker, src, italics, size, symbol = symbol)
```

Reply:
Did not work when I tried it before submitting the PR, the message would fade away the next second and the symbol not show up. I'm guessing named arguments just don't work in this case.

Reply:
Fair enough